docs/reference/search/search-your-data/sort-search-results.asciidoc

```diff
@@ -423,7 +422,7 @@ GET /_search
 
 `distance_type`::
 
-    How to compute the distance. Can either be `arc` (default), or `plane` (faster, but inaccurate on long distances and close to the poles).
+    Deprecated, how to compute the distance, this is handled internally.
```

Comment:
As suggested above, I'd leave this documentation as is and add a deprecation admon.